syntax = "proto3";

package opencog_services;

message Command {
   string cmd = 1;
   repeated string params = 2;
}

message CommandOutput {
   string output = 1;
}

service OpencogServices {
   rpc Execute(Command) returns (CommandOutput) {}
   rpc AsynchronousTask(Command) returns (CommandOutput) {}
}